mkpkg

```diff
@@ -158,6 +158,12 @@ class Functions:
                 continue
         return sorted(deps)
 
+    def mapfiles(self, src, dst, *bins, **kwargs):
+        suffix = self.pkg.vars['suffix'] if 'append_suffix' in kwargs and kwargs['append_suffix'] else ''
```

Comment:
When the expression gets a bit complicated another option is just use a classical `if` statement:
```py
suffix = ""
if 'append_suffix' in kwargs and kwargs['append_suffix']:
    suffix = self.pkg.vars['suffix']
```

But this is probably a shorter and more clear alternative:

```py
suffix = kwargs.get('append_suffix', "")
```

:)